include/arch/arm/cortex_m/mpu/nxp_mpu.h

```diff
@@ -26,8 +26,16 @@
 #define MPU_REGION_SU    ((3 << 3) | (3 << 9) | (3 << 15) | (3 << 21))
 
 /* Some helper defines for common regions */
+#if defined(CONFIG_MPU_ALLOW_FLASH_WRITE)
+/* Required to initialize the flash driver */
+#define REGION_RAM_ATTR (MPU_REGION_READ | MPU_REGION_WRITE | \
+			 MPU_REGION_EXEC |  MPU_REGION_SU)
```

Comment:
I think we should bring more attention to the fact that not only are we enabling flash write, but also execution from ram. Can you mention this explicitly in the commit message? Is there a better config name to indicate this?